app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3 and command.split()[0] == "mv":
+        command, source_file_name, new_file_info = command.split()
```

Comment:
Why did you name the variables so differently? `source_file_name` and `new_file_info`?
Use `source_file_name` and `target_file_name`(for example) instead to keep consistency